demos/image-classification/index.js

```diff
@@ -466,6 +467,24 @@ const changeImage = async () => {
   await main();
 };
 
+const setupORT = async () => {
+  const ortversion = document.querySelector("#ortversion");
+  removeElement("onnxruntime-web");
+  let ortVersion = "1.19.0-dev.20240621-69d522f4e9";
+  let ortLink = "";
+  if (ortVersion && ortVersion.length > 4) {
```

Comment:
What is the meaning of `> 4`? Does that mean an ORT release of `1.20` (4 characters) would pick the older local build, even though it's newer than `1.19.0`?

Reply:
This function was used in Web AI Benchmark to check ORT Web builds from public, and even internal built by @Honry. I will make it simpler for Image Classification.